File: backend/models/PageModel.go
package models

import "database/sql"

type AppPage struct {
	Id               uint    `json:"id"`
	Page_title       sql.NullString `json:"page_title"`
	Page_description sql.NullString `json:"page_description"`
	Path             sql.NullString  `json:"path"`
	Redirect         sql.NullString `json:"redirect"`
	Title            sql.NullString `json:"title"`
	Content          sql.NullString `json:"content"`
	Layout_filename  sql.NullString `json:"layout_filename"`
	Scripts          sql.NullString `json:"scripts"`
	Styles           sql.NullString `json:"styles"`
}

File: admin/src/app/modules/pages/models/PageModel.ts
export interface PageInterface {
    id: string
    created_at: Date
    updated_at: Date
    title: string
    path: string
    redirect: string
    page_title: string
    page_description: string
    content: string
    layout_filename: string
    scripts: string
    styles: string
    hidden: boolean
}
